mandatory/1-fix-functions.js

```diff
@@ -10,31 +10,26 @@
 
 */
 
-function mood() {
+function mood(a) {
   let isHappy = true;
 
-  if (isHappy) {
+  if (a === true) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
-
-  if (isBigEnough) {
+  if (num > 10) {
     return "num is greater than 10";
   } else {
     return "num is not big enough";
   }
 }
 
 function get3rdIndex(arr) {
-  let index = 3;
-  let element;
-
-  return element;
+ return arr[3];
```

Comment:
I would recommend not modifying the original function too much 
```suggestion
function get3rdIndex(arr) {
  let index = 3;
  let element =arr[index];

  return element;
}
```